### PR TITLE
Update MenuRoutines.cs

### PR DESCRIPTION
**Pull request by tuday2**
*Fixed Issue #35  to allow inputs > 1 digit
*Cleaned up If Statements from custom ANSI files PR
*Fixed crash caused by negative number input to menu

### CODE REVIEW

File: MBBSEmu/HostProcess/HostRoutines/MenuRoutines.cs [hunk 1]
```diff
@@ -308,18 +306,29 @@ private void MainMenuInput(SessionBase session, Dictionary<string, MbbsModule> m
                 return;
             }
 
-            var inputValue = Encoding.ASCII.GetString(session.InputBuffer.ToArray()).ToUpper()[0];
+            var inputValue = Encoding.ASCII.GetString(session.InputBuffer.ToArray());
+            
+            //convert to uppercase, trim
+            var inputCommand = inputValue.ToUpper().TrimEnd('\0');
 
+            //Exit if command longer then 2 characters (up to 99 modules) or a negative number
```
Comment (paladine) [inline MBBSEmu/HostProcess/HostRoutines/MenuRoutines.cs:314]: s/then/than

File: MBBSEmu/HostProcess/HostRoutines/MenuRoutines.cs [hunk 2]
```diff
@@ -308,18 +306,29 @@ private void MainMenuInput(SessionBase session, Dictionary<string, MbbsModule> m
...
+            if (inputCommand.Length > 2 || inputCommand.Contains('-'))
```
Comment (paladine) [inline MBBSEmu/HostProcess/HostRoutines/MenuRoutines.cs:315]: This entire section might not be necessary. 

Leave the inputCommand == "X" logic, and then relying on `Int.TryParse` should be sufficient no? Probably have to add an additional check on selectedMenuItem >= 0

**Review by paladine — CHANGES_REQUESTED** [review]

File: MBBSEmu/HostProcess/HostRoutines/MenuRoutines.cs (hunk 1 above)
Comment (tuday2) [reply MBBSEmu/HostProcess/HostRoutines/MenuRoutines.cs]: removed.

File: MBBSEmu/HostProcess/HostRoutines/MenuRoutines.cs (hunk 2 above)
Comment (tuday2) [reply MBBSEmu/HostProcess/HostRoutines/MenuRoutines.cs]: Added OR statement with >= 0 and removed extra check. Only outstanding item is you can type 00000000 and get the 0 menu item... not sure it matters but that's the only other thing that slips through

Comment (enusbaum) [reply MBBSEmu/HostProcess/HostRoutines/MenuRoutines.cs]: I'm fine with 00000 still resolving to 0

**Comment by enusbaum** [comment]
Fixes #35

**Pull request merged**